src/main/java/racingcar/step3/RacingCar.java

```diff
@@ -0,0 +1,25 @@
+package racingcar.step3;
+
+public class RacingCar {
+    private final int car;
+    private int[] cars;
+    private final int number;
+
+    public RacingCar(int car, int number) {
+        this.car = car;
+        this.number = number;
+    }
+
+    public int getRacingCarInput(){
+        return car;
+    }
+
+    public int[] getRacingCars(){
+        cars = new int[car];
+        return cars;
+    }
```

Comment:
```java
    public RacingCar(int car, int number) {
        this.car = car;
        this.number = number;
        this.cars = new int[car]; //추가
    }
```
생성자에서 이렇게 추가를 하고
```suggestion
    public int[] getCars(){
        return cars;
    }
```
이렇게 변경해 보는건 어떨까요?